lib/java/com/google/android/material/appbar/MaterialToolbar.java

```diff
@@ -97,6 +112,13 @@ public MaterialToolbar(@NonNull Context context, @Nullable AttributeSet attrs, i
     titleCentered = a.getBoolean(R.styleable.MaterialToolbar_titleCentered, false);
     subtitleCentered = a.getBoolean(R.styleable.MaterialToolbar_subtitleCentered, false);
 
+    final int index = a.getInt(R.styleable.MaterialToolbar_logoScaleType, 3);
+    if (index >= 0) {
```

Comment:
I think we should check upper bound instead. I think index is always larger than or equal to 0 here.